src/main/java/teammates/storage/search/SearchQuery.java

```diff
@@ -40,14 +43,31 @@ public int getFilterSize(){
     }
     
     protected SearchQuery setTextFilter(String textField, String queryString){
-        String sanitizedQueryString = Sanitizer.sanitizeForSearch(queryString).toLowerCase().trim();
+        String sanitizedQueryString;
+        if(isValidEmailAddress(queryString)){
```

Comment:
Please put a space before and after the parenthesis.
`if (isValidEmailAddress(queryString)) {`